src/inline.d

```diff
@@ -35,8 +35,10 @@ import ddmd.tokens;
 import ddmd.visitor;
 
 private:
-enum LOG = false;
-enum CANINLINE_LOG = false;
+
+enum LOG = 0;
+enum CANINLINE_LOG = 0;
+enum EXPANDINLINE_LOG = 0;
```

Comment:
To switch the debug prints, rewriting `0` to `1`, or vice versa, is lower cost than rewiring between `true` and `false`.